trident/data/line_lists/lines.txt

```diff
@@ -129,7 +129,7 @@ Si II          1808.013000    2.540000e+06    2.490000e-03
 Si II          1309.276000    4.730000e+08    1.130000e-01     Si II* 1309
 Si II          1304.370000    3.040000e+09    1.090000e+00                
 Si II          1264.738000    8.300000e+08    1.940000e-01     Si II* 1265
-Si II          1260.422000    4.030000e+08    1.880000e-01                
+Si II          1260.422000    2.950000e+09    1.180000e+01                
```

Comment:
Here's what I've found from [NIST](https://physics.nist.gov/cgi-bin/ASD/lines1.pl?spectra=Si+II&limits_type=0&low_w=1260&upp_w=1261&unit=0&de=0&java_window=3&java_mult=&format=0&line_out=0&en_unit=0&output=0&bibrefs=1&page_size=15&show_obs_wl=1&show_calc_wl=1&unc_out=1&order_out=0&max_low_enrg=&show_av=2&max_upp_enrg=&tsb_value=0&min_str=&A_out=0&f_out=on&S_out=on&intens_out=on&max_str=&allowed_out=1&forbid_out=1&min_accur=&min_intens=&conf_out=on&term_out=on&enrg_out=on&J_out=on&submit=Retrieve+Data): 2.57e+09 | 1.22e+00

Reply:
Weird--how did we get so far off?  I thought @devinsilvia pulled these straight from NIST, but maybe the NIST values have changed?

Reply:
Upon further investigation, the two lines specified in this PR were misidentified by the script that queried NIST values and populated this list.  It appears at first look that there are no other differences between the line list and true NIST values.  I will continue to check this and report back.

Reply:
I happen to be looking at this value. I'm comparing with the [mortan03 table](https://github.com/linetools/linetools/blob/master/linetools/data/lines/morton03_table2.dat), and find there's a typo: 1.18e+01 should be 1.18E+00.

Reply:
@qobilidop, can you issue a PR for that?

Reply:
@brittonsmith, sure, will do.